wfe2/wfe.go

```diff
@@ -876,11 +845,10 @@ func (wfe *WebFrontEndImpl) revokeCertByKeyID(
 		// Try to find a stored final certificate for the serial number
 		serial := core.SerialToString(parsedCertificate.SerialNumber)
 		cert, err := wfe.SA.GetCertificate(ctx, serial)
-		if errors.Is(err, berrors.NotFound) && features.Enabled(features.PrecertificateRevocation) {
-			// If there was an error, it was a not found error, and the precertificate
-			// revocation feature is enabled, then try to find a stored precert.
-			pbCert, err := wfe.SA.GetPrecertificate(ctx,
-				&sapb.Serial{Serial: serial})
+		if errors.Is(err, berrors.NotFound) {
+			// If there was an error, it was a Not Found error, then maybe they're
```

Comment:
For consistency with the other comments in this package.
```suggestion
			// If there was an error, it was a not found error, then maybe they're
```